CONTRIBUTING.md

```diff
@@ -155,10 +139,9 @@ introduced as a boundary node.
 
 Each of the nodes should satisfy at least one learning objective.
 
-In order to develop learning goals, faculty should answer the question, “What
+In order to develop learning goals, contributors should answer the question, “What
```

Comment:
I've been using the Bloom's taxonomy action objectives for the learning objectives meta data. So I don't know if you wanted people to draw on a common source for consistency.

https://www.apu.edu/live_data/files/333/blooms_taxonomy_action_verbs.pdf

Reply:
There are some other links below - the second has a list of verbs... are they different?

Reply:
It looks like the link to the UC-Berkeley site has the same verbs. I think the link I provided is more straightforward and direct. The action verbs are right there and the definition from Bloom is provided for each category. Regardless of what link to choose, I recommend that everyone draw from a common set of action verbs, so we're all in the same ballpark.